### ISSUE TITLE
Free/Libre replacement fonts.  

### ISSUE THREAD

**Issue by clort81** [calref/cboe#228]
My research into the fonts used for openboe indicate they seem to be restricted by copyright.

To help avoid any possible issues going forward, I am developing replacements for bold, plain and maidenworld TTF fonts.

While the current TTF fonts are rendered by SFML to a close approximation to the original, the antialiasing causes a 'style clash' from a UI design perspective.  Some people are more sensitive to such things than others:

![BoE-ClortFont](https://user-images.githubusercontent.com/51511737/72258814-48158280-360f-11ea-8db8-54a87455a629.png)
IF your browser scales the image, you will not see the proper 1:1 bitmapped view - click on image and save it to view at 1:1. 
(And yes, the character names need to be moved up 1 or 2 px)

SFML fails to add an extra pixel of space when 'bolding' the font, but maybe this is something i overlooked in font creation - an extra pixel can be added with
```
str_to_draw.setLetterSpacing(2.0); //Clort test a wider kerning
```
in "gfx/render_text.cpp" 94 (right after options.style.applyTo(str_to_draw), but this widens all fonts.  It would need to only add a pixel space for the bold font, if we choose to adopt it, or add it as an option.

It requires a point size of 11px, so references to BOLD font with 10 or 12px need to be changed to 11.

![clortfont](https://user-images.githubusercontent.com/51511737/72259127-fb7e7700-360f-11ea-97a7-1c382b97d59c.png)

My replacement fonts are freely redistributable under the BSD license.

**Comment by clort81** [comment]
The kerning issue with SFML bold fonts seems to occur with other bdf fonts (those which load at all) as well:
![fonts-sfml_kerning](https://user-images.githubusercontent.com/51511737/72343975-7313db00-36d0-11ea-8648-afa5daa19fc9.png)

ClortFont-11 kerns fine when bolding is off (see console at lower right):
![fonts-clortconsole](https://user-images.githubusercontent.com/51511737/72344023-8fb01300-36d0-11ea-9d55-eea475014f3e.png)

ClortFont-11 for 'plain' font is closer to original WinDOS BoE font.  Whether it's preferable to openboe's existing 'plain.ttf' is probably a matter of personal taste.  I'd be happy with either.

**Comment by clort81** [comment]
I've finished a 20-point replacement bitmap font for 'MaidenWord'.  It consists of 100% hand drawn glyphs by me, Clort.  Currently available as .pcx and .bdf under BSD license.
![clortdamsel1 1b](https://user-images.githubusercontent.com/51511737/72498483-c486ac00-382f-11ea-86f4-996774679760.png)

I have not finished patching OBoE to use the bitmap font, but a friend helped me convert it to TTF, which SFML can scale to the various sizes the game currently demands.
![clortdamsel_boe_ttf](https://user-images.githubusercontent.com/51511737/72498613-12031900-3830-11ea-8c17-b4f0273c46f2.png)

The results with scaled ClortDamsel TTF are not better than MaidenWord, but at least we now have license-unencumbered fonts for the game.  I will continue work until results look good.

**Comment by CelticMinstrel** [comment]
I'm closing this since the posted fonts no longer seem to be available, and issue #317 now tracks the underlying questions of aliasing and whether we're even allowed to use the fonts we're using.
